### DISCUSSION TITLE
What is the name of the feature available in nightly Rust that allows read signals to be functions instead of needing .get()?

### DISCUSSION THREAD

**Discussion by GabrielBarbosaGV** [leptos-rs/leptos#1481] [Q&A] [answered]
Good day to all!

I am aware of the need for nightly Rust if we are to make use of this feature, but I do not know how it works. I've been searching for it with "Nightly Rust function call syntax", but didn't hit answers. Could someone point me to an explanation regarding the feature?

Many thanks for the great library and the help!

**Comment by gbj** [comment, marked as answer]
`fn_traits` and `unboxed_closures` https://github.com/leptos-rs/leptos/blob/248beb4a55375c7852b0669cafd424105836a40f/leptos_reactive/src/lib.rs#L2-L3

**Reply by GabrielBarbosaGV** [reply]
Thank you very much!
